challenges/prototypes.js

```diff
@@ -6,28 +6,44 @@
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
 
+let CuboidMaker = (length, width, height) => {
+  this.length = length;
+  this.width = width;
+  this.height = height;
+}
+
 
 /* == Step 2: Volume Method ==
   Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
   
   Formula for cuboid volume: length * width * height
 */
 
+CuboidMaker.prototype.findVolume = () => {
+  return this.length * this.width * this.height;
+}
 
 /* == Step 3: Surface Area Method ==
   Create another method using CuboidMaker's prototype that returns the surface area of a given cuboid's length, width, and height. 
 
   Formula for cuboid surface area of a cube: 2 * (length * width + length * height + width * height)
 */
 
+CuboidMaker.prototype.findSurfaceArea = () => {
+  return 2 * (this.length * this.width + this.length * this.height + this.width * this.height);
+}
+
 
 /* == Step 4: Create a new object that uses CuboidMaker ==
   Create a cuboid object that uses the new keyword to use our CuboidMaker constructor
   Add properties and values of length: 4, width: 5, and height: 5 to cuboid.   
 */
 
+let cuboid = new CuboidMaker(4, 5, 5);
```

Comment:
Elyssia here you have to create your cuboid object. As of right now you are only giving 3 values but you are not creating an object or assign value to each parameter. Like so ...
```
const cuboid = new CuboidMaker({
    length: 4,
    width: 5,
    height: 5,
});
```